src/V3Tristate.cpp

```diff
@@ -412,6 +412,34 @@ class TristateVisitor final : public TristateBaseVisitor {
         }
         return VN_AS(invarp->user1p(), Var);
     }
+    AstConst* getEnpConstp(AstConst* constp) {
```

Comment:
```suggestion
    AstConst* getEnConstp(AstConst* constp) {
```
I think the `p` at the end is enough? Even that might not be necessary.

Reply:
Done. I updated https://github.com/verilator/verilator/pull/3545.